site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -86,7 +86,7 @@ paths:
                application/json:
                   schema:
                      $ref: '#/components/schemas/Tenant'
-            required: true
+            required: false
```

Comment:
I think this should be false for the case where tenant id is supplied as well

Reply:
Also, we have the same case for the `devices` endpoint. I would make it consistent between `tenants` and `devices`, but keep it required for the `credentials`. @sophokles73  wdyt?

Reply:
+1 for keeping it consistent, i.e. do not require a body for devices as well. However, that could go into a separate PR.

Reply:
Sounds good. Let's wrap tenants in this one and tackle devices in the separate one

Reply:
@dejanb I have aligned this for when an ID is supplied. I've also updated the `devices` section to match. 
Creating a device is now as simple as : 
`curl -i -X POST http://localhost:28080/v1/devices/DEFAULT_TENANT`